wimm.Secundatives.UnitTests/Extensions/MappingExtensions_Test.cs

```diff
@@ -48,7 +44,7 @@ public async Task MapMaybe_ValueAndFuncReturnsTask_ReturnsValueTask()
         }
 
         [Fact]
-        public void MapMaybe_NoneAndFuncReturnsMaybe_ReturnsNone()
+        public void MapMaybe_NoneAndFuncReturnsMaybeWithValue_ReturnsNone()
```

Comment:
I wonder why we named these `FuncReturnsMaybe`. Not a suggestion here `FuncReturnsSome` seems like the mroe accurate term (and more concise than `FuncReturnsMaybeWithValue`